app/services/live.js

```diff
@@ -30,6 +30,9 @@ export default class LiveService extends Service {
   @globalRef('videoEl') videoEl;
   @tracked peers;
   @tracked isScreenShareOn;
```

Comment:
i think it should have a default value as false

Reply:
yes it should be but I have not written this code